contents/docs/user-guides/paths.mdx

```diff
@@ -6,26 +6,192 @@ showTitle: true
+-   Understand which parts of your application people are actually using
+-   Identifying points where users are getting confused or are unable to find what they are looking for
+-   Find out why users aren't discovering new features
 
-## Filters
+## Getting started
 
-In order to determine patterns between groups of users who may behave similarly, you can filter paths according to your preferred criteria. PostHog has several built-in filters, including OS, Browser, Time, and IP address. Filters are accessed within the main 'User Paths' tab under Insights. 
+### Creating a new Paths insight
+
+To create a new Paths insight, first navigate to the 'Insights' section using the side menu and then click the 'New insight' button at the top of the screen. By default, new insights that are created will be set to Trends, so we'll click on the drop down and switch the type to 'Paths'
+
+![switching the event type](../../images/docs/user-guides/paths/switching-type.png)
+
+We've now created our first Paths insight!
+
+![example paths](../../images/docs/user-guides/paths/example.png)
+
+Depending on the amount of data you've ingesting into PostHog, the graph you see may be quite complex, so we'd highly recommend continuing to the next section to get a better idea of how to understand and customize your paths visualization.
+
+### Understanding paths
+
+Paths are one of PostHog's most powerful tools for exploring how users journey through your app.
+However, they can be somewhat overwhelming when first getting started with them, simply due to the complexity and density of the visualization.
+This section provides a bit of background to help understand what paths are at a basic level, and how you can use them to discover things about your user journey.
+
+There are a couple of terms that will be handy to define beforehand:
+
+-   **Step** - This is a single point along a user's journey, and corresponds with the events that were sent during a user's session. For example, each page a user viewed or event they send will be a separate step.
+-   **Path** - This is the basic unit of the paths insight, and corresponds to a user _moving from one step to another_
+
+A step can be any [Event](/docs/user-guides/actions) or [Action](/docs/user-guides/actions) (such as pageviews, button clicks, or a combination of both).
+
+### Looking at a simple path
+
+Now that we know the basic concepts in a path, let's take a look at some simple examples.
+The basic concept of a path is most easily seen by setting the number of steps above the visualization to only 2.
+For simplicity, we'll also only be thinking about paths of Page views, but the thought process for custom events is exactly the same.
+
+![path visualization with only 2 steps](../../images/docs/user-guides/paths/2-step.png)
+
+Here, I've set the [start point](#changing-the-start-and-end-point) to be the Docs homepage for PostHog, and I've [limited the number of paths](#maximum-number-of-paths) to only 5.
```

Comment:
```suggestion
Here, I've set the [start point](#changing-the-start-and-end-point) to be the Docs homepage for PostHog, and I've [limited the number of paths](#maximum-number-of-paths) to only 2.
```

Reply:
It's definitely slightly confusing with the difference between steps and paths, but in this case I limited the number of paths to 5 while I kept the number of steps at 2, so I think this should still be 5.

Reply:
D'oh, gotcha!